README.md

```diff
@@ -19,21 +33,21 @@ AWS credentials can be exported as environment variables or permissions can be g
 |backup_dir| Path to directory which will hold the final backup|
 |s3_bucket| S3 bucket to store/retrieve buckets to/from|
 |s3_path| S3 Path prefix, can be used for backing up multiple core nodes to the same bucket|
+|gpg_key| GPG key ID used for signing and verification of the stellar-core backups. The provided ID is the Stellar signing key|
 
 ## Usage As Command Line Tool
 
-##### backup
+##### backup but don't shasum the backup files or gpg sign the SHA256SUMS file
 
 ```
-stellar-core-backup --config /etc/stellar/stellar-core-backup.conf --backup
+stellar-core-backup --config /etc/stellar/stellar-core-backup.conf --backup --no-verify
```

Comment:
I think the examples we provide should use verification/signing but explain in a subsection that this can be disabled using the `--no-verify` for both the backup/restore process.